BDArmory/UI/BDArmorySetup.cs

```diff
@@ -3439,6 +3439,8 @@ void WindowSettings(int windowID)
                             BDArmorySettings.SF_GRAVITY = true;
                             BDArmorySettings.SF_DRAGMULT = 30;
                         }
+                        //TODO - convert these to gamemode types - e.g. Firerate Increase on Kill, Rapid Deployment, Spacemode, etc, and clear out all the empty round values
```

Comment:
Will there be a special setup for the mutators for `BDArmorySettings.RUNWAY_PROJECT_ROUND == 61` here once logins finally decides on it or will that be provided via some cfg file?

Reply:
.cfg file.